app/views/pages/secondary-question-banks.html.erb

```diff
@@ -0,0 +1,66 @@
+
+<%= render 'components/hero', hero_title: 'Secondary question banks', colour: 'lime-green-bg', full_width: true %>
+<div class="govuk-width-container">
+  <div class="govuk-main-wrapper govuk-!-padding-bottom-3">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+        <p class="govuk-body"><strong>
+          This collection contains quality-assured question banks which can be used with Key Stage 3 computing and GCSE Computer Science students.
+        </p></strong>
+        <p class="govuk-body">
+          The collection includes a series of 10 topics, covering areas across the computing curriculum. Each topic area includes two sets of questions, setup as both Google and Microsoft Forms, allowing you to duplicate into your own account/s to use with your students.
+        </p>
+      </div>
+
+      <div class="govuk-grid-column-one-third">
+        <%= render SecondaryQuestionBanksComponent.new(
+          text: 'Use the links below to duplicate the forms into your own Google/Microsoft account. Once duplicated, they will be shareable with your students, with results only viewable by you.',
+          link_title: '',
+          tracking_event_category: 'Courses',
+          tracking_event_label: 'Bursaries',
+          bottom_margin: false
+        ) %>
+      </div>
+    </div>
+  </div>
+</div>
+
+<div class="govuk-width-container">
+	<div class="govuk-main-wrapper govuk-!-padding-top-0 govuk-!-padding-bottom-0">
+		<div class="govuk-grid-row">
+				<div class="govuk-grid-column-full">
+				  <h4 class="govuk-heading-m">Topics</h4>
+            <ul class="govuk-list govuk-body govuk-!-padding-bottom-4 govuk-!-padding-top-2">
+              <% t('pages.secondary_question_banks').each do |resource|%>
+                <li class="curriculum__list--item">
+                    <h2 class="govuk-heading-s"><%= resource[:title] %></h2>
+                    <span class="govuk-!-display-inline-block govuk-!-padding-right-9">
+                      <span class="govuk-!-padding-right-6 govuk-!-font-weight-bold">Google Form:</span> 
+                      <span class="govuk-!-padding-right-6"><%= link_to "Set A", "#{resource[:google_form_set_a_link]}", class: "govuk-link ncce-link"%></span> 
+                      <span class="govuk-!-padding-bottom-0"><%= link_to "Set B", "#{resource[:google_form_set_b_link]}", class: "govuk-link ncce-link" %></span>
+                    </span>
+                    <span class="govuk-!-display-inline-block">
+                      <span class="govuk-!-padding-right-6 govuk-!-font-weight-bold">Microsoft Form:</span> 
+                      <span class="govuk-!-padding-right-6"><%= link_to "Set A", "#{resource[:microsoft_form_set_a_link]}", class: "govuk-link ncce-link"%></span> 
+                      <span><%= link_to "Set B", "#{resource[:microsoft_form_set_b_link]}", class: "govuk-link ncce-link" %></span>
+                    </span>
+                </li>
```

Comment:
this might make a good view component or a partial. YMMV

Reply:
resolved thank you